src/components/availability-panel/drag-drop-context/index.tsx

```diff
@@ -87,8 +87,13 @@ const DragDropcontext: FC<dragDropProps> = ({
         const assignee = result.combine.droppableId === 'tasks'
           ? result.draggableId
           : result.combine.draggableId;
+        const dateObject:any = new Date();
+        const startedOn:any = dateObject / 1000;
+        const endsOn:number = startedOn + (14 * 86400);
```

Comment:
Let's put `86400` and `14` in a file-level global variable

Reply:
Got it